zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -1528,21 +1528,20 @@ impl<F: Field> ExecutionConfig<F> {
         let assigned_stored_expressions = self.assign_stored_expressions(region, offset, step)?;
 
         // enable with `CHECK_RW_LOOKUP=true`
-        if *CHECK_RW_LOOKUP && verbose {
-            let is_padding_step = matches!(step.execution_state, ExecutionState::EndBlock)
-                && step.rw_indices.is_empty();
-            if !is_padding_step {
-                // expensive function call
-                Self::check_rw_lookup(
-                    &assigned_stored_expressions,
-                    offset,
-                    step,
-                    call,
-                    transaction,
-                    block,
-                    region.challenges(),
-                );
-            }
+        // if *CHECK_RW_LOOKUP && verbose {
+        let is_padding_step =
```

Comment:
use CHECK_RW_LOOKUP

Reply:
restore` if *CHECK_RW_LOOKUP && verbose` in  [7340a21](https://github.com/scroll-tech/zkevm-circuits/pull/466/commits/7340a21bd5f5e030e2ef1658cea7003e5a7c3cd7)